extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileWriter.java

```diff
@@ -27,13 +28,14 @@
 import static org.apache.parquet.format.Util.writePageHeader;
 
 public class ParquetFileWriter {
-    private static ParquetMetadataConverter metadataConverter = new ParquetMetadataConverter();
+    private static final ParquetMetadataConverter metadataConverter = new ParquetMetadataConverter();
     private static final int VERSION = 1;
 
     private final SeekableByteChannel writeChannel;
     private final MessageType type;
     private final int pageSize;
     private final ByteBufferAllocator allocator;
+    @ReferentialIntegrity
```

Comment:
Cloned from DHE - we will need it (...as much as dhe does) if we remove the unused addRowGroup method

Reply:
I don't see any reason in the DHC codebase why we would keep that reference if we delete the unused `addRowGroup` method. It's not needed for referential integrity of anything I can find.
I slightly want to annotate that method `@SuppressWarnings("unused")`, rather than delete it. It's illustrating a pattern we don't use right now, but might consider some day.